src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -90911,84 +90878,92 @@ INSERT INTO tab_event_conditions VALUES(5938,'EVENT','ParticleBinDissolvedEventU
+INSERT INTO tab_known_transporters VALUES('SLC22A5','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A6','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A6','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A7','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A8','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A8','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A9','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC47A1','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLCO1A1','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1A2','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1A3','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLCO1A4','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B2','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B3','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO2B1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO4C1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC47A2','Human','Efflux');
+CREATE TABLE tab_known_transporters_containers
+(
+    gene           text,
+    species        text,
+    container_type text,
+    container_name text,
+    transport_type text,
+    membrane       text,
+    constraint tab_known_transporters_containers_pk
+        primary key (gene, species, container_type, container_name),
+    constraint tab_known_transporters_containers_tab_known_transporters_gene_species_fk
+        foreign key (gene, species) references tab_known_transporters (gene, species)
+);
+INSERT INTO tab_known_transporters_containers VALUES('ABCC3','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC3','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC4','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC4','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC6','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('ABCC6','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('SLC22A9','Human','COMPARTMENT','Pericentral','Efflux','Basolateral');
+INSERT INTO tab_known_transporters_containers VALUES('SLC22A9','Human','COMPARTMENT','Periportal','Efflux','Basolateral');
+CREATE TABLE tab_transport_directions (transport_direction text CONSTRAINT tab_transport_directions_pk PRIMARY KEY, display_name text NOT NULL, description text NOT NULL, icon text DEFAULT Influx);

```

Comment:
`...icon text DEFAULT Influx`
The default value for the icon does not really make sense here, right?

Reply:
This was from before. No idea what this is.